Recipes/Recipes.API/Models/Entities/RecipeIngredient.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Recipes.API.Models.Entities;
+public class RecipeIngredient
```

Comment:
Wait, didn't we decide that we won't have separate/standalone entities for ingredients [here](https://github.com/BenasB/vu-psk/issues/8#issuecomment-2098901522)? And no many-to-many

Reply:
Changed to simple list. I guess we are not doing quantities then.

Reply:
Quantities will be a part of ingredients, like in the examples we discussed before, e.g. an ingredient could be "3 garlic cloves" or "Minced meat 500g"